script/CreateObolValidatorManagerScript.s.sol

```diff
@@ -13,27 +15,34 @@ import {ObolValidatorManagerFactory} from "../src/ovm/ObolValidatorManagerFactor
 //   --rpc-url https://rpc.pectra-devnet-5.ethpandaops.io/ --broadcast "<factory_address>"
 //
 contract CreateObolValidatorManagerScript is Script {
-    function run(address deployedOWRV2Factory) external {
+    function run(address deployedFactory) external {
         uint256 privKey = vm.envUint("PRIVATE_KEY");
+        address deployerAddress = vm.addr(privKey);
+
+        console.log("Deployer address: ", deployerAddress);
 
         vm.startBroadcast(privKey);
 
-        ObolValidatorManagerFactory factory = ObolValidatorManagerFactory(deployedOWRV2Factory);
+        ObolValidatorManagerFactory factory = ObolValidatorManagerFactory(deployedFactory);
 
-        address owner = msg.sender;
-        address recoveryAddress = msg.sender;
-        address principalRecipient = msg.sender;
-        address rewardsRecipient = msg.sender;
+        address owner = deployerAddress;
+        address recoveryAddress = deployerAddress;
+        address principalRecipient = deployerAddress;
+        address rewardsRecipient = deployerAddress;
         uint64 principalThreshold = 16 ether / 1 gwei;
```

Comment:
We'll need to make a version of this script for real use where someone running the script can set external addresses as owners, principals, recipients, etc.